accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/RequestParameters.java

```diff
@@ -0,0 +1,60 @@
+package biz.netcentric.cq.tools.actool.ui;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.commons.lang3.StringUtils;
+
+import biz.netcentric.cq.tools.actool.api.AcInstallationService;
+import biz.netcentric.cq.tools.actool.impl.AcInstallationServiceImpl;
+
+public class RequestParameters {
+
+    static RequestParameters fromRequest(HttpServletRequest req, AcInstallationService acInstallationService) {
```

Comment:
I would move this to AcToolUiService (i.e. the abstract base class for both Web Console and Touch UI Console) and use dedicated getters.